shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2652,23 +2665,47 @@ \subsubsection{Transaction Fees}
 \label{transaction-fees-1}
 
 All transaction fees from all transactions from all blocks created
-during the epoch will be used as rewards.
+during the epoch will be added to the rewards pot of that epoch.
 
 \subsubsection{Deposits}
 \label{deposits-1}
 
 As explained in \cref{fees-deposits}, a part of the deposits for
 certificates and UTxO entries are non-refundable, and contribute to
-the rewards pool instead.
+the rewards pot instead.
 
 \subsubsection{Monetary Expansion}
 \label{monetary-expansion}
 
-Let \(T\) be the total amount of ADA in existence during a specific
-epoch, and let \(T_\infty\) be the maximal possible amount of ADA in the
-future. At this moment, \(T=31,000,000,000\) and
-\(T_\infty=45,000,000,000\). Then the amount of
-\(\rho\cdot(T_\infty - T)\) ADA will be newly created.
+Every epoch, the total amount of ada in circulation \(T\) will be increased by
+adding ada to the rewards pot. To ensure the amount of ada never exceeds a
+finite, specified limit \(T_\infty\), the increase will reduce exponentially.
+Also, the increase will depend on the number of blocks that have been produced
+during the epoch. Specifically, each epoch, the contribution from monetary
+expansion to the rewards pot is given by
+\[\min(\eta, 1) \rho \left(T_\infty - T\right)\,,\]
```

Comment:
Just to clarify. What is the purpose of `min(η,1)`? Is it possible that created number of block is *larger* than expected number of blocks in an epoch? Is it because of slots with no leaders in Praos? Do I get it right that nodes will know the expected approximate number of leader-less slots?

Reply:
Exactly! In Praos, you can have more blocks in an epoch than you would expect on average.